src/applications/rated-disabilities/components/Learn.jsx

```diff
@@ -18,6 +18,7 @@ export default function Learn() {
       <va-link
         href="/disability/about-disability-ratings"
         text="About VA disability ratings"
+        uswds="false"
```

Comment:
`va-link` doesn't have a `uswds` property.

Reply:
yes so even though it doesnt have it yet, we still wanted to add it and set it to false since all components are supposed to be updated to V3 by feb 16.